crates/mun_syntax/src/grammar.ron

```diff
@@ -48,6 +48,7 @@ Grammar(
         ["%=", "PERCENTEQ"],
         ["..=", "DOTDOTEQ"],
         ["::", "COLONCOLON"],
+        ["->", "MINUSGT"],
```

Comment:
I think `THIN_ARROW` might be a better name.

Reply:
Okay, done!